ui/src/components/AccessPermission.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import axios, {AxiosRequestConfig, AxiosResponse} from "axios";
+
+
+interface Props {
+}
+
+interface State {
+    checked: boolean;
+    hasError: boolean;
+    errorMessage: string;
+}
+
+class AccessPermission extends React.Component<Props, State> {
+
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            checked: false,
+            hasError: false,
+            errorMessage: ''
```

Comment:
Looks like `errorMessage` isn't used anywhere

Reply:
It gets populated by error message of axios response . But not displayed in UI . I am thinking to keep it, if we want to make UI to populate with different error message according to file parse error then we can use it .